openedx/core/djangoapps/plugin_api/views.py

```diff
@@ -78,6 +77,43 @@ def add_fragment_resource_urls(self, fragment):
         for js_file in self.js_dependencies():
             fragment.add_javascript_url(staticfiles_storage.url(js_file))
 
+    def create_base_standalone_context(self, request, fragment, **kwargs):
+        """
+        Creates the base context to use when rendering a standalone page.
+        """
+        return {
+            'uses_pattern_library': True,
+            'disable_accordion': True,
+            'allow_iframing': True,
+            'disable_header': True,
+            'disable_footer': True,
+            'disable_window_wrap': True,
+        }
+
+    def _add_studio_context_variables(self, request, context):
+        """
+        Add additional context variables needed by Studio.
```

Comment:
I've pushed up changes that go with what you had, but try to keep the first sentence under 70 characters.